airsonic-main/src/main/java/org/airsonic/player/dao/TranscodingDao.java

```diff
@@ -72,11 +72,15 @@
      * @param playerId       The player ID.
      * @param transcodingIds ID's of the active transcodings.
      */
-    public void setTranscodingsForPlayer(String playerId, int[] transcodingIds) {
-        update("delete from player_transcoding2 where player_id = ?", playerId);
-        String sql = "insert into player_transcoding2(player_id, transcoding_id) values (?, ?)";
-        for (int transcodingId : transcodingIds) {
-            update(sql, playerId, transcodingId);
+    public void setTranscodingsForPlayer(Integer playerId, int[] transcodingIds) {
+        if (playerId == null) {
+            return;
```

Comment:
Hm, this seems like we should ensure we aren't calling this method with a null playerId rather than adding a check here.

Reply:
Agreed - will search where it is called from and see if I can add checks there.